loopout_test.go

```diff
@@ -15,6 +15,105 @@ import (
 	"github.com/lightninglabs/loop/test"
 )
 
+// TestLoopOutPayment tests the first part of the loop out process up to the
+// point where the off-chain payments are made.
+func TestLoopOutPayment(t *testing.T) {
```

Comment:
`TestLoopOutPaymentSplit`? The current name is a bit vague.

Reply:
It doesn't really test the split itself, but really just the payment parameters. Renamed to `TestLoopOutPaymentParameters`